js_modules/dagit/packages/core/src/runs/RunDetails.tsx

```diff
@@ -112,14 +113,38 @@ export const RunDetails: React.FC<{
   );
 };
 
+
 export const RunConfigDialog: React.FC<{run: RunFragment; isJob: boolean}> = ({run, isJob}) => {
   const [showDialog, setShowDialog] = React.useState(false);
   const {rootServerURI} = React.useContext(AppContext);
   const runConfigYaml = yaml.stringify(run.runConfig) || '';
+
+  const launchpadPath = () => {
+    const path = `/playground/setup-from-run/${run.id}`;
+
+    if (run.repositoryOrigin) {
+      return workspacePipelinePath({
+        repoName: run.repositoryOrigin.repositoryName,
+        repoLocation: run.repositoryOrigin.repositoryLocationName,
+        pipelineName: run.pipelineName,
+        isJob,
+        path,
+      });
+    }
+
+    return workspacePipelinePathGuessRepo(run.pipelineName, isJob, path);
+  };
```

Comment:
Can this be extracted out into a utility that can be used both here and in the existing `Open in Playground…` location?